cmd/zrepl/zrepl.c

```diff
@@ -28,49 +28,6 @@ static void uzfs_zvol_io_ack_sender(void *arg);
 kthread_t	*conn_accpt_thread;
 kthread_t	*uzfs_timer_thread;
 kthread_t	*mgmt_conn_thread;
-char 		*pool_name = NULL;
-struct 		in_addr addr = {0};
-int zrepl_import(int argc, char **argv);
-int zrepl_start(int argc, char **argv);
-
-typedef struct zrepl_command {
-	const char *cmd_name;
-	int (*func)(int, char **);
-} zrepl_cmd_t;
-
-static zrepl_cmd_t cmd_table[] = {
-	{"import",	zrepl_import},
-	{"start",	zrepl_start},
-	{NULL},
-};
-
-#define	NCMDS   (sizeof (cmd_table) / sizeof (zrepl_cmd_t))
-
-int
-find_command(const char *cmd_name, int *index)
-{
-	for (int i = 0; i < NCMDS; i++) {
-		if (cmd_table[i].cmd_name == NULL)
-			continue;
-		if (strcmp(cmd_name, cmd_table[i].cmd_name) == 0) {
-			*index = i;
-			return (0);
-		}
-	}
-	return (1);
-}
-
-void
-help(void)
-{
-	/*
-	 * XXX need to do better here
-	 */
-
-	printf("zrepl command args ... \nwhere 'command' is one of:\n\n");
-	printf("\t import <pool_name> [-t ip address)]\n");
-	printf("\t start [-t ip address)]\n");
-}
```

Comment:
Now when we removed start and import subcommands and zrepl starts without any parameters, I have a question what is the expected way of importing a pool holding replicas by sidecar. Is it:
```
cmd/zrepl/zrepl
# after zrepl is up and running (probably execute in loop with sleep 1):
cmd/zpool/zpool import <name>
```
?

Reply:
yes @jkryl